.eslintrc.json

```diff
@@ -54,6 +54,7 @@
     "@typescript-eslint/prefer-interface": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/no-non-null-assertion": "off",
+    "@typescript-eslint/no-unused-vars": "off",
```

Comment:
Our linter complains about unused vars when we import something we use only for typing (e.g. `MuiTabs` in `Tab.tsx`). I disabled the rule temporarily before we can fix this in a more proper manner.